orientdb/src/main/java/com/yahoo/ycsb/db/OrientDBClient.java

```diff
@@ -48,13 +48,6 @@
 /**
  * OrientDB client for YCSB framework.
  * 
- * Properties to set:
- * 
- * orientdb.url=local:C:/temp/databases or remote:localhost:2424 <br>
- * orientdb.database=ycsb <br>
- * orientdb.user=admin <br>
- * orientdb.password=admin <br>
- * 
  * @author Luca Garulli
```

Comment:
I'd prefer not; I don't know if anyone else feels strongly about it.

Essentially, I think of author tags as a "ask this person if there are issues" and I'd prefer we get folks in the habit of asking the community rather than a named individual.